upmc-enterprises/cda2fhir
# Clinical status fixes

#### What does this PR do?

Creates clinicalStatus based on date for conditions and allergies rather than containing status.

#### Related JIRA tickets:

UPMCFHIR-157
UPMCFHIR-158

#### How should this be manually tested?

Run the new tests, also run the validation on the gold file and see if the errors resurface.

#### Background/Context:

Had to revise logic to satisfy FHIR3 requirement.

#### New JIRA tickets for clinical review required?

No, one already logged.
